internal/util/projutil/project_util.go

```diff
@@ -207,11 +207,31 @@ func GetOperatorType() OperatorType {
 	return OperatorTypeUnknown
 }
 
+// PluginKeyToOperatorType converts a plugin key string to an operator project
+// type.
+// TODO(estroz): this can probably be made more robust by checking known
+// plugin keys directly.
+func PluginKeyToOperatorType(pluginKey string) OperatorType {
+	switch {
+	case strings.HasPrefix(pluginKey, "go"):
+		return OperatorTypeGo
+	case strings.HasPrefix(pluginKey, "helm"):
+		return OperatorTypeHelm
+	case strings.HasPrefix(pluginKey, "ansible"):
+		return OperatorTypeAnsible
+	}
+	return ""
+}
+
+// IsOperatorGo will return true when the type of the project is Go
 func IsOperatorGo() bool {
-	// todo: in the future we should check the plugin prefix to ensure the operator type
-	// for now, we can assume that any project with the kubebuilder layout is Go Type
+	// If the project has the new layout we will check the type in the config file
 	if kbutil.HasProjectFile() {
-		return true
+		cfg, err := kbutil.ReadConfig()
+		if err != nil {
+			log.Print(err, "Error reading config")
```

Comment:
This should be
```suggestion
			log.Fatalf("Error reading config: %v")
```
Ditto below.